### PR TITLE
[12.0] [MIG] mgmtsystem nonconformity hr

### PR DESCRIPTION
**Pull request by mymage**

### CODE REVIEW

**Comment by mymage** [comment]
@ivantodorovich Ping for a fast review (migration) :-)

File: mgmtsystem_nonconformity_hr/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name":     "Management System - Nonconformity HR",
+    "summary":  "Bridge module between hr and mgmsystem and",
+    "version":  "12.0.1.0.0",
+
+    "author":   "Associazione PNLUG - Gruppo Odoo, Odoo Community Association (OCA)",
+    "website":  "https://gitlab.com/PNLUG/Odoo/management-system-improvements/tree/"
+                "11.0/mgmtsystem_extended",
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_hr/__manifest__.py]:
```suggestion
    "website":  "https://github.com/OCA/management-system",
```

File: mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MgmtsystemMgmHR(models.Model):
+    """
+    Extend nonconformity adding fields for department
+    """
+
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py]:
```suggestion
```

self explainatory ;) no need for comments

File: mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    _inherit = ['mgmtsystem.nonconformity']
+
+    # new fields
+    # department reference
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py]:
```suggestion
```
self explainatory ;) no need for comments

File: mgmtsystem_nonconformity_hr/readme/CONFIGURE.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+This module doesn't need configuration
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_hr/readme/CONFIGURE.rst]: Remove this field

File: mgmtsystem_nonconformity_hr/readme/INSTALL.rst [hunk 5]
```diff
@@ -0,0 +1 @@
+This module will be automatically installed
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_hr/readme/INSTALL.rst]: Remove this file

File: mgmtsystem_nonconformity_hr/views/mgmtsystem_nonconformity_views.xml [hunk 6]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- Copyright 2019 Marcelo Frare (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     Copyright 2019 Stefano Consolaro (Associazione PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl) -->
+     
+<odoo>
+
+    <!-- inherit NC form view 
+            adding department -->
```
Comment (ivantodorovich) [inline mgmtsystem_nonconformity_hr/views/mgmtsystem_nonconformity_views.xml]:
```suggestion
```
self explainatory ;) no need for comments

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
Minor comments, otherwise LGTM :+1: thanks!!

File: mgmtsystem_nonconformity_hr/__manifest__.py (hunk 1 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/__manifest__.py]: Ok

File: mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py (hunk 2 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py]: I like comments for newbie, anyway ok.

File: mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py (hunk 3 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/models/mgmtsystem_nonconformity_hr.py]: like before, ok

File: mgmtsystem_nonconformity_hr/readme/INSTALL.rst (hunk 5 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/readme/INSTALL.rst]: ok

File: mgmtsystem_nonconformity_hr/views/mgmtsystem_nonconformity_views.xml (hunk 6 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/views/mgmtsystem_nonconformity_views.xml]: like before, ok

**Review by ivantodorovich — APPROVED** [review]

File: mgmtsystem_nonconformity_hr/__init__.py [hunk 7]
```diff
@@ -0,0 +1,4 @@
+
```
Comment (dreispt) [inline mgmtsystem_nonconformity_hr/__init__.py]:
```suggestion
```

File: mgmtsystem_nonconformity_hr/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name":     "Management System - Nonconformity HR",
+    "summary":  "Bridge module between hr and mgmsystem and",
```
Comment (dreispt) [inline mgmtsystem_nonconformity_hr/__manifest__.py]:
```suggestion
    "summary":  "Bridge module between HR and Non Conformities",
```

File: mgmtsystem_nonconformity_hr/__manifest__.py [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+    "category": "Management System",
+    "depends": [
+        'hr',
+        'mgmtsystem_nonconformity',
+        ],
+    "data": [
+        'views/mgmtsystem_nonconformity_views.xml',
+        ],
+    'application': False,
```
Comment (dreispt) [inline mgmtsystem_nonconformity_hr/__manifest__.py]:
```suggestion
```

**Review by dreispt — APPROVED** [review]
I wonder if this would be better named ` mgmtsystem_nonconformity_hr_department`, since it follows what was done [here](https://github.com/OCA/department).
Code review looks good, left a few sugegstions.

**Comment by ivantodorovich** [comment]
> I wonder if this would be better named ` mgmtsystem_nonconformity_hr_department`, since it follows what was done [here](https://github.com/OCA/department).
> Code review looks good, left a few sugegstions.

Well, actually it depends on `hr` core module, so I think the name is accurate :)

**Comment by mymage** [comment]
> I wonder if this would be better named ` mgmtsystem_nonconformity_hr_department`, since it follows what was done [here](https://github.com/OCA/department).

@dreispt For v 11.0 it was merged as mgmtsystem_nonconformity_hr. Can we change the name for v 12.0+? Must we change the name on the v 11.0?

File: mgmtsystem_nonconformity_hr/__manifest__.py (hunk 9 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/__manifest__.py]: Ok

File: mgmtsystem_nonconformity_hr/__manifest__.py (hunk 8 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/__manifest__.py]: Thanks

File: mgmtsystem_nonconformity_hr/__init__.py (hunk 7 above)
Comment (mymage) [reply mgmtsystem_nonconformity_hr/__init__.py]: Ok

**Comment by ivantodorovich** [comment]
I wouldn't change the module name, to me it's ok
This module is not really related to https://github.com/OCA/department, it's just a glue module with the core `hr` module

@dreispt do you think this is blocking, or can we merge?

**Pull request merged**